atk/src/auto/enums.rs

```diff
@@ -211,6 +211,23 @@ pub enum RelationType {
     __Unknown(i32),
 }
 
+impl RelationType {
+    pub fn for_name(name: &str) -> RelationType {
+        assert_initialized_main_thread!();
+        unsafe { from_glib(ffi::atk_relation_type_for_name(name.to_glib_none().0)) }
+    }
+
+    pub fn get_name(self) -> Option<glib::GString> {
+        assert_initialized_main_thread!();
+        unsafe { from_glib_none(ffi::atk_relation_type_get_name(self.to_glib())) }
+    }
+
+    pub fn register(name: &str) -> RelationType {
```

Comment:
That seems problematic and will probably need manual bindings to be properly usable.